Server/ReasnAPI/ReasnAPI/Services/TagService.cs

```diff
@@ -74,20 +74,30 @@ public void DeleteTag(int tagId)
         context.SaveChanges();
     }
 
-    public void ForceDeleteTag(int tagId)
+    public void ForceDeleteTag(TagDto tagDto)
     {
-        var tag = context.Tags.FirstOrDefault(r => r.Id == tagId);
+        if (tagDto is null)
+        {
+            throw new ArgumentException("No tag provided");
+        }
```

Comment:
So, here you check for null, sometimes you don't, please unify it, for example in method above you don't check that

Reply:
added a check where its missing